### ISSUE TITLE
Remove tuple stubs

### ISSUE THREAD

**Issue by lthls** [ocaml-flambda/ocaml#195]
This means storing information about the tuplified status of functions as part of `Exported_code.Calling_convention`, and reading this information when inlining (so that the tuple argument is properly unboxed) and when creating Cmm closures (so that the arity is correctly set as negative, and `caml_tuplifyN` is used instead of `caml_curryN`).

**Comment by lthls** [comment]
Example of function allocating a stub, from `kb.ml`:
```ocaml
let compsubst subst1 subst2 = 
  (List.map (fun (v,t) -> (v, substitute subst1 t)) subst2) @ subst1
```

**Comment by Gbury** [comment]
Solved by #201
